src/main/java/org/jabref/gui/SendAsGeneralEmailAction.java

```diff
@@ -0,0 +1,38 @@
+package org.jabref.gui;
+
+import java.io.StringWriter;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+import org.jabref.gui.actions.ActionHelper;
+import org.jabref.preferences.PreferencesService;
+
+/**
+ * Sends the selected entries to any specifiable email
+ * by populating the email body
+ */
+public class SendAsGeneralEmailAction extends SendAsEMailAction {
```

Comment:
I don't like `General`. Maybe `Standard` is better? --> `SendAsStandardEmailAction`